src/js/DataTables/DataTable.js

```diff
@@ -225,6 +236,14 @@ export default class DataTable extends PureComponent {
       </table>
     );
 
-    return responsive ? <div className="md-data-table--responsive">{table}</div> : table;
+    return responsive ? (
+      <div className="md-data-table--responsive" style={{width}}>
+        <div className="md-data-table--responsive-header-placeholder">
+          <div className="md-data-table--responsive-vertical-scroll" style={{height: height - 56}}>
```

Comment:
`56` is the height of the header... I have no idea on how to avoid this hardcoding.